Workbooks/SecurityStatus.json

```diff
@@ -110,7 +106,7 @@
             "crossComponentResources": [
               "{Subscriptions}"
             ],
-            "value": "/subscriptions/82931e73-05c6-4da8-a666-bc4a7dd1bd3e/resourceGroups/fabrikamltdprodrg/providers/Microsoft.OperationalInsights/workspaces/fabrikamltdprod",
+            "value": "/subscriptions/44e4eff8-1fcb-4a22-a7d6-992ac7286382/resourceGroups/SOC/providers/Microsoft.OperationalInsights/workspaces/CyberSecurityDemo",
```

Comment:
please change the subscription, resource group and workspace to the following generic values:
"/subscriptions/<subs_ID>/resourcegroups/<rg_name>/providers/microsoft.operationalinsights/workspaces/<workspace_name>"